### PR TITLE
Cleanup registries support

### PR DESCRIPTION
**Pull request by pdalpra**
- Support pushing to prod registry
- Drop support for other registries than dev in other cases

### CODE REVIEW

**Comment by pdalpra** [comment]
ping

File: cmd/docker/tag-and-push.go [hunk 1]
```diff
@@ -14,14 +14,16 @@ var (
...
 )
 
 var tagAndPushCmd = &cobra.Command{
 	Use:   "tag-and-push",
 	Short: "Tags and pushes an image to the Docker registry in a single operation",
 	Long: `tag-and-push both tags and pushes an image to Teads Docker registry in a single operation.
 The name of the source (and target) image to push is specified by the '--image' flag, and expects that
-the built image has been initially tagged with 'teads-central vars hash' (~ HASH environment variable).
+The built image has been initially tagged with 'teads-central vars hash' (~ HASH environment variable).
+The registry to tag/push images to is specified by the '--registry' flag, defaulting to the dev registry.
```
Comment (gbougeard) [inline cmd/docker/tag-and-push.go]: it's more an option than a flag, isn't it?

Comment (pdalpra) [inline cmd/docker/tag-and-push.go]: indeed, fixed !

**Comment by gbougeard** [comment]
LGTM

**Pull request merged**